assets/components/patronsEvents/patronsEvents.jsx

```diff
@@ -32,11 +41,18 @@ export default function PatronsEvents() {
         heading="Live events"
         copy="Meet Guardian journalists and readers at our events, debates, interviews and festivals"
         ctaText="Find out more"
-        ctaUrl="https://membership.theguardian.com/events"
+        ctaUrl={getMemLink('events', props.campaignCode)}
         ctaId="live-events"
         ctaAccessibilityHint="Find out more about Guardian live events"
       />
     </PageSection>
   );
 
 }
+
+
+// ----- Default Props ----- //
```

Comment:
To avoid jumping around the file, I would move default props to the top of the file (below props definition)

Reply:
You can't, that's a use before definition.